### ISSUE TITLE
"an error occured" flash message without error in logs 

### ISSUE THREAD

**Issue by cecilehabran** [craftcms/cms#6032]
### Description
I get an error message "An error occured" almost everytime I save an entry. I can't find any error in the logs nor any information in the debug bar.

I installed a fresh version to make sure it wasn't some messed up files in the vendor, everything is up to date. 
I have 3 sites in the same group, different languages.

Where should I look ?

### Additional info

- Craft version:Craft Pro 3.4.18
- PHP version:7.2.30
- Database driver & version:MySQL 5.6.46
- Plugins & versions:
Field Manager | 2.1.5
Freeform | 3.6.7
Neo | 2.7.9.1
Redactor | 2.6.1
Super Table | 2.4.9

**Comment by brandonkelly** [comment]
Have your browser’s Network tab open before saving the entry, and when the error shows up, check if there are any non-2xx XHR requests logged in there.

**Comment by cecilehabran** [comment]
No, nothing, only 200 XHR that went ok, and no error in the console.

I can't find any real issue (I can save the entries even if the error message is displayed) 
except that a user with full permissions appears to have some difficulties when trying to access to assets or save an entry created from another user - but I don't think that is related.

**Comment by brandonkelly** [comment]
Any chance you can give us admin + SSH/FTP access to the site? If so please send credentials over to support@craftcms.com.

**Comment by cecilehabran** [comment]
Thanks, I'll do that.
